samples/unit-tests/series/marker-clusters/demo.js

```diff
@@ -824,3 +824,60 @@ QUnit.test('OptimizedKmeans algorithm tests.', function (assert) {
         'optimizedKmeans should use kmeans again when extremes range is greater than on chart init.'
     );
 });
+
+QUnit.test('Maps tests.', function (assert) {
```

Comment:
Can we combine these assertions with an existing map chart to avoid increasing the processing time of the test suite?

And if not, can we load a lighter map, without proj4 and with simpler and fewer shapes?